content/docs/command-reference/exp/pull.md

```diff
@@ -0,0 +1,62 @@
+# exp pull
+
+Download a single [experiment](/doc/command-reference/exp) to a Git remote, and
+its data to a `dvc remote`.
+
+## Synopsis
+
+```usage
+usage: dvc experiments pull [-h] [-q | -v] [-f] [--no-cache]
+                            [-r <name>] [-j <number>] [--run-cache]
+                            git_remote experiment
+
+positional arguments:
+  git_remote          Git remote name or repo URL
+  experiment          Experiment to pull
```

Comment:
I noticed the CLI uses `<>` around these names. Can we avoid that for consistency? Currently those are only used for metavalues.

- [ ] I should actually make a full list of core updates to sync up to these docs...